app/Silex/Metadata/MetadataServiceProvider.php

```diff
@@ -8,29 +8,38 @@
 use Broadway\EventDispatcher\CallableEventDispatcher;
 use Broadway\EventSourcing\MetadataEnrichment\MetadataEnrichingEventStreamDecorator;
 use CultuurNet\UDB3\CommandHandling\ResqueCommandBus;
+use CultuurNet\UDB3\Container\AbstractServiceProvider;
 use CultuurNet\UDB3\EventSourcing\LazyCallbackMetadataEnricher;
 use CultuurNet\UDB3\CommandHandling\ContextFactory;
-use CultuurNet\UDB3\Silex\Container\HybridContainerApplication;
-use Silex\Application;
-use Silex\ServiceProviderInterface;
 
-final class MetadataServiceProvider implements ServiceProviderInterface
+final class MetadataServiceProvider extends AbstractServiceProvider
 {
-    public function register(Application $app)
+    protected function getProvidedServiceNames(): array
     {
-        $app['context'] = null;
+        return [
+            'metadata_enricher',
+            'event_stream_metadata_enricher',
+            'command_bus_event_dispatcher',
+        ];
+    }
+
+    public function register(): void
+    {
+        $container = $this->getContainer();
+
+        $container->addShared('context', '');
```

Comment:
I've changed this from null to empty string. Because adding a null causes different behaviour in the PSR-Container vs the Silex Application.